### ISSUE TITLE
Stop automatically adding ingredients as labels: the natural flavors label

### ISSUE THREAD

**Issue by October-Food-Facts** [openfoodfacts/openfoodfacts-server#10716]
Ingredients go in the ingredients section, labels go in the labels section.
- A `Natural flavors` label is automatically added based on the ingredients. Possibly, a few other labels also exist like this. Each ingredient generating a label is a problem and hence the bug categorization.
- Following this approach, every single ingredient is in need of its own label. `Whole cereals`, `Animal rennet`, `Non-animal rennet`, `Bovine gelatine`, `Processed seaweeds`, `Added proteins`, `Added vitamins`, `Added fibers`, `Added starch`, `Modified starch`, `Buffalo milk`, `Skimmed milk`, `Pasteurized milk`, `Whole milk`, `Sea salt`, `Table salt`, `Added salt`, `High oleic oil`, `Reconstituted milk powder`, `Gluten-free oats`, `Extra virgin olive oil` (caned tuna), `Refined oils`, `Unrefined sugar`, `Dried tomatoes`, `Edam cheese`, `Mozzarella cheese`, `Anhydrous milk fat matter`, `Red wine`, `White wine`, `Mineral water`, `Organic corn`, `Wild-caught salmon`. There's a never-ending list of ingredients with properties and all of them will be dumped as labels.

Please, for the love of God, don't put the ingredients list on the labels field forcibly and automatically.
Labels are put by the producers on the packages, standing on their own, outside of the ingredients list. Those are the ones that will be noted down, correspondingly. It can't be done automatically, not by duplicating the ingredients list in the labels field.

**Comment by aleene** [comment]
I agree that not all adjectives should become labels automatically. I wonder where is this automatic assignment occurs.

**Comment by benbenben2** [comment]
Do you have some examples @October-Food-Facts of labels that should not be parsed from the ingredients list?

It is also very common to find labels in the ingredients list.


When ingredient is not recognized, the code checks if it is a label. One error for natural flavors would be when the label exists for a language in the label taxonomy but not in the ingredients taxonomy, for example.

**Comment by October-Food-Facts** [comment]
Do you know where the list of labels that is automatically assigned based on the ingredients list can be consulted? I think I only know specifically of `natural flavors` being unduly assigned.

**Comment by October-Food-Facts** [comment]
The idea is that when `aromas naturais` or `aroma natural` is an ingredient in the ingredients list:
the label `Sabores naturais` in the labels list is NOT automatically generated and enforced anymore as it currently is.

Once that is done and if that's accomplished, this issue will be closed and solved. :)

**Comment by teolemon** [comment]
- It's probably Robotoff that adds anything that is a label. 
- It happens that Natural Flavor is a label
- We should check whether we have a Robotoff side label blocklist to disable that behaviour
- Actually we have an allowlist, and natural flavors is not on it: https://github.com/openfoodfacts/robotoff/blob/main/data/ocr/label_whitelist.txt

**Comment by teolemon** [comment]
@October-Food-Facts Can you link to an example product where this happened ?

**Comment by October-Food-Facts** [comment]
[All products](https://pt-en.openfoodfacts.org/facets/labels/natural-flavors) that are saved with `aroma natural` written in the ingredients list automatically get a natural flavors label in the labels list. It's unrelated to robot-off and can be easily tested.

**Comment by benbenben2** [comment]
Seems to be fixed @October-Food-Facts 

![Image](https://github.com/user-attachments/assets/9dfb649d-1295-4208-8da3-fa05322ce416)
![Image](https://github.com/user-attachments/assets/2d798f51-cf3b-4bb5-8adb-2e86c316544a)

**Comment by October-Food-Facts** [comment]
I seem to now be able to save pages without the label being added automatically.
